docs/fundamentals/systems/overlays/configuration.md

```diff
@@ -1,7 +1,7 @@
 # Systems >> Overlays >> Configuration ||40
 
 ```js script
-import { html } from '@mdjs/mdjs-preview';
+import { html as previewHtml } from '@mdjs/mdjs-preview';
```

Comment:
why not use one html per page?

Reply:
Because then i can't load html as part of the example.